src/adb.js

```diff
@@ -1 +1,353 @@
-// https://github.com/appcelerator/titanium_mobile/blob/master/node_modules/titanium-sdk/lib/adb.js
+import 'babel-polyfill';
+import 'source-map-support/register';
+import fs from 'fs';
+import path from 'path';
+import wrench from 'wrench';
+import { spawn } from 'child_process';
+
+import { Connection } from './connection';
+import { Emulator, EmulatorManager } from './emulator';
+import * as util from './util';
+import * as sdk from './sdk';
+
+/**
+ * Debug flag that is enabled via the android.debugadb setting.
+ */
+let DEBUG = false;
+
+/**
+ * Creates an ADB object.
+ * @class
+ * @classdesc Provides methods to interact with the Android Debug Bridge (ADB).
+ * @constructor
+ */
+export class ADB {
+	constructor(config) {
```

Comment:
`config` should become a plain old JavaScript object `opts`. Besure to default it to an empty object:

    `constructor(opts = {}) {`